doc/en/how-to/capture-warnings.rst

```diff
@@ -42,8 +42,18 @@ Running pytest now produces this output:
     -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
     ======================= 1 passed, 1 warning in 0.12s =======================
 
-The ``-W`` flag can be passed to control which warnings will be displayed or even turn
-them into errors:
+Controlling warnings
+--------------------
+
+Similar to Python's `warning filter` and :option:`-W option <python:-W>` flag, pytest provides
```

Comment:
As seen at the docs preview, this link isn't working:
https://pytest--9464.org.readthedocs.build/en/9464/how-to/capture-warnings.html#controlling-warnings

An underscore should fix it:

https://docutils.sourceforge.io/docs/user/rst/quickref.html#implicit-hyperlink-targets

```suggestion
Similar to Python's `warning filter`_ and :option:`-W option <python:-W>` flag, pytest provides
```

Reply:
Done, good catch!